google/data_source_google_container_engine_versions.go

```diff
@@ -53,23 +58,46 @@ func dataSourceGoogleContainerEngineVersionsRead(d *schema.ResourceData, meta in
 		return err
 	}
 
-	zone, err := getZone(d, meta.(*Config))
-	if err != nil {
-		return err
+	var location string
```

Comment:
could you use `getLocation(d, config)` instead of this block, or does it have a reason to behave differently than the cluster resource?

Reply:
nope, didn't know about that - will change it